server/settings/components/common.py

```diff
@@ -205,3 +208,13 @@
 
 # Timeouts
 EMAIL_TIMEOUT = 5
+
+# Authbroker
+AUTHBROKER_URL = "https://sso.trade.gov.uk/"
+AUTHBROKER_CLIENT_ID = "B9xGM4WWRuQ2eYTALaV3RTS8AV4beSyTnTSk6tO1"
+AUTHBROKER_CLIENT_SECRET = "d5Yv2YijSSMj0VNxUZ570JEjVpKh5yzqmkHcWixALz5MsR0seCwn2GP1DO3fPqbSma1jBpuIkHdKeIlKHGk5PFDb6D6qU1uWDlgu5r0sfQJtTEw4BEkonTOuFWdDJayG"
```

Comment:
```suggestion
AUTHBROKER_CLIENT_SECRET = config('AUTHBROKER_CLIENT_SECRET')
```
probably best to access indirectly through `config` and set in .env

Reply:
🤦‍♂